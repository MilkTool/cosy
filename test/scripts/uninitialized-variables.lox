var a = "1"
print a

var b
b = "2"
print b

var c
print c // runtime error: uninitialized variable
