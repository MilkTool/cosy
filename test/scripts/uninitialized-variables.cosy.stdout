1
2
[line 9] Runtime Error: Accessing uninitialized variable "c".
